### PR TITLE
Switch to using port 3000 for the app in both development and CI

### PR DESCRIPTION
**Pull request by frankieroberto**
This updates the keycloak configuration and the `docker-compose` files to use port 3000 for the main application.

The main advantage of this is allowing developers to run the main application outside of Docker without having to remember to specify a different port from the Rails default one.

### CODE REVIEW

**Comment by slorek** [comment]
@frankieroberto should be able to close this now as #1375 and #1382 address these issues.

**Comment by frankieroberto** [comment]
@slorek yep, although I think we still need to figure out how to avoid needing the changes in `keycloak/configuration/initial-setup.json` locally?

**Comment by slorek** [comment]
You can add an extra redirect URL to include port 3000, but the admin URL looks like it can only have one. I guess you could just make sure you run it on port 3002...

**Comment by slorek** [comment]
@frankieroberto we can probably make use of your changes to `keycloak/configuration/initial-setup.json` since ports 3000/3001 were reserved for `psd-web` in the Docker context I believe. These ports should now be available for use as we have now removed PSD from this repo. The benefit being you won't need to specify port 3002 anymore. Do you want to rebase this branch so we can try it?

**Comment by frankieroberto** [comment]
@slorek can you spot why CI might be failing the system tests? Must be connected with the port change somehow but I can't see it.

**Comment by slorek** [comment]
@frankieroberto the system tests will spin up a new server listening on port 3001 (was 3003) so in the Keycloak config where you changed the whitelisted URLs from 3003 you should change them to 3001. Also check that 3003 isn’t configured/hard coded anywhere.

**Comment by frankieroberto** [comment]
@slorek I’ve whitelisted 3001 in the keycloak config, but the system tests are still failing. I did think I’d updated the docker-compose files to run the app on port 3000 (line 5 of `cosmetics-web/spec/system_helper.rb`) but that doesn't seem to be helping either... 😞

File: cosmetics-web/spec/system_helper.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 require "selenium-webdriver"
 
 ENV["HTTP_HOST"] = "localhost"
-ENV["HTTP_PORT"] = "3003"
+ENV["HTTP_PORT"] = "3000"
```
Comment (slorek) [inline cosmetics-web/spec/system_helper.rb]: I think this should be `3001`

**Comment by frankieroberto** [comment]
@slorek this is still not working... 😞  Any ideas?  It'd be nice to get to a point where we can run `bin/rails` without having any uncommitted changes to the keycloak config.

**Comment by frankieroberto** [comment]
Closing this for now, as we have a reasonable workaround.

**Pull request closed without merging**